rosalind_cons.py

```diff
@@ -4,10 +4,12 @@ def main(data):
     lines = [text.replace("\n", "") for text in data.split(">")]
     columns = zip(*[text[13:] for text in lines if text != lines[0]])
     consensus = "".join([Counter(column).most_common(1)[0][0] for column in columns])
+    data = {'A': '', 'C': '', 'G': '', 'T': ''}
 
-    a = " ".join([str(Counter(column)['A']) for column in columns])
-    c = " ".join([str(Counter(column)['C']) for column in columns])
-    g = " ".join([str(Counter(column)['G']) for column in columns])
-    t = " ".join([str(Counter(column)['T']) for column in columns])
+    for column in columns:
+        data['A'] += ' ' + str(Counter(column)['A'])
+        data['C'] += ' ' + str(Counter(column)['C'])
+        data['G'] += ' ' + str(Counter(column)['G'])
+        data['T'] += ' ' + str(Counter(column)['T'])
 
-    return '%s\nA: %s\nC: %s\nG: %s\nT: %s' % (consensus, a, c, g, t)
+    return '%s\nA:%s\nC:%s\nG:%s\nT:%s' % (consensus, data['A'], data['C'], data['G'], data['T'])
```

Comment:
Using `%s` too much like this can be improved with Python's [`format`](https://docs.python.org/2/library/string.html#format-examples)